mkdocs.yml

```diff
@@ -102,6 +109,7 @@ nav:
           - STAR-CCM+:  Documentation/Applications/starccm.md
           - VASP:       Documentation/Applications/vasp.md
           - WRF:        Documentation/Applications/wrf.md
+          - Xpress:     Documentation/Applications/xpressmp.md
```

Comment:
Just leaving a note -- Xpress also isn't in the applications table.

Reply:
Added now.